<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FaceAttendence</class>
 <widget class="QMainWindow" name="FaceAttendence">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FaceAttendence</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#widget_3
{
	
	background-color: rgb(94, 94, 94);
}

QWidget#widget_2
{
	background-color: rgba(180, 180, 180, 63);
}

QLabel#lb_2
{
	font:25 16pt &quot;微软雅黑 Light&quot;;
	border:none;
	color:rgb(255,255,255);
}
/*设置姓名等*/
QLabel[name = &quot;key&quot;]
{
	background-color:#20232A;
	font:20 12pt&quot;微软雅黑 Light&quot;;
	color:#808183;
}
/*设置姓名的值等*/
QLabel[name = &quot;value&quot;]
{
	background-color:#20232A;
	font:20 12pt&quot;微软雅黑 Light&quot;;
	color:#ffffff;
}
/*设置识别成功人脸的圆形*/
QLabel#lb_headpic
{
	background-color:#20232A;
	border-radius:70px;
}
/*设置标题*/
QLabel#lb_title
{
	font:20 20pt&quot;微软雅黑 Bold&quot;;
	color:#ffffff;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>480</width>
      <height>480</height>
     </rect>
    </property>
    <widget class="QWidget" name="wg_success" native="true">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>350</y>
       <width>181</width>
       <height>60</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="lb_1">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="../res.qrc">:/res/yes.png</pixmap>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lb_2">
        <property name="text">
         <string>认证成功</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="lb_traceFace">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>271</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../res.qrc">:/res/crc.png</pixmap>
     </property>
    </widget>
    <widget class="QLabel" name="lb_camera">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>480</width>
       <height>480</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <zorder>lb_camera</zorder>
    <zorder>wg_success</zorder>
    <zorder>lb_traceFace</zorder>
   </widget>
   <widget class="QWidget" name="widget_3" native="true">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>0</y>
      <width>320</width>
      <height>480</height>
     </rect>
    </property>
    <widget class="QLabel" name="lb_headpic">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>70</y>
       <width>160</width>
       <height>160</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>160</width>
       <height>166</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="lb_title">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>281</width>
       <height>51</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>80</height>
      </size>
     </property>
     <property name="text">
      <string>人脸识别考勤系统</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="widget_4" native="true">
     <property name="geometry">
      <rect>
       <x>1</x>
       <y>242</y>
       <width>311</width>
       <height>241</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>工号</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="name" stdset="0">
         <string>key</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="lb_employeeID">
        <property name="text">
         <string/>
        </property>
        <property name="name" stdset="0">
         <string>value</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#20232A;</string>
        </property>
        <property name="text">
         <string>所属部门</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLabel" name="label_5">
        <property name="styleSheet">
         <string notr="true">background-color:#20232A;</string>
        </property>
        <property name="text">
         <string> </string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>姓名</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="name" stdset="0">
         <string>key</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="lb_nickname">
        <property name="text">
         <string/>
        </property>
        <property name="name" stdset="0">
         <string>value</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_2">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#20232A;</string>
        </property>
        <property name="text">
         <string>性别</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QLabel" name="label_7">
        <property name="styleSheet">
         <string notr="true">background-color:#20232A;</string>
        </property>
        <property name="text">
         <string> </string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>岗位</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="name" stdset="0">
         <string>key</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="lb_address">
        <property name="text">
         <string/>
        </property>
        <property name="name" stdset="0">
         <string>value</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>微软雅黑 Light</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color:#20232A;</string>
        </property>
        <property name="text">
         <string>人员类型</string>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QLabel" name="label_9">
        <property name="styleSheet">
         <string notr="true">background-color:#20232A;</string>
        </property>
        <property name="text">
         <string> </string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_10">
        <property name="maximumSize">
         <size>
          <width>60</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>时间</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignmentFlag::AlignCenter</set>
        </property>
        <property name="name" stdset="0">
         <string>key</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="lb_time">
        <property name="text">
         <string/>
        </property>
        <property name="name" stdset="0">
         <string>value</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../res.qrc"/>
 </resources>
 <connections/>
</ui>
